tensorflow/python/keras/callbacks.py

```diff
@@ -1308,12 +1308,11 @@ def on_epoch_end(self, epoch, logs=None):
 @keras_export('keras.callbacks.LearningRateScheduler')
 class LearningRateScheduler(Callback):
   """Learning rate scheduler.
-
-  Arguments:
-      schedule: a function that takes an epoch index as input
-          (integer, indexed from 0) and returns a new
-          learning rate as output (float).
-      verbose: int. 0: quiet, 1: update messages.
+  
+  A callback for scheduling the learning rate to change after every epoch or batch.
```

Comment:
The learning rate doesn't necessarily change after every epoch/batch, as it depends on the `schedule` function. Also I don't think it currently supports changing at batch boundaries.

Reply:
@rchao spot on! How about we change it to:
```
A callback for scheduling to update the optimizer's learning rate at the start of each epoch. The learning rate used in this callback replaces the default learning rate set by the gradient descent optimization algorithm.
```
I am just trying to make sure people do less googling when working the callbacks APIs. Unfortunately, this was not in @fchollet 's excellent Deep Learning with Python with but is covered in @ageron's Hands-On ML with SKlearn, Keras and TF (upcoming v2 of the book).

Reply:
Thanks for the reply!